src/main/java/uk/gov/hmcts/reform/judicialapi/elinks/service/impl/ElinksPeopleServiceImpl.java

```diff
@@ -337,14 +331,15 @@ private void saveRoleDetails(String personalCode, List<RoleRequest> judiciaryRol
                     .endDate(convertToLocalDateTime(roleRequest.getEndDate()))
                     .personalCode(personalCode)
                     .jurisdictionRoleId(roleRequest.getJudiciaryRoleId())
+                    .jurisdictionRoleNameId(roleRequest.getJudiciaryRoleNameId())
                     .build());
             } catch (Exception e) {
-                log.warn("Role type  not loaded for " + personalCode);
+                log.warn("Judicial additional role  not loaded for " + personalCode);
                 partialSuccessFlag = true;
                 elinkDataExceptionHelper.auditException(JUDICIAL_REF_DATA_ELINKS,
                     now(),
-                    personalCode,
```

Comment:
is there any requirement to remove personal code from the record. please cross check this with pradipta